packages/toolpad-core/src/PageContainer/PageContainer.test.tsx

```diff
@@ -112,4 +112,44 @@ describe('PageContainer', () => {
 
     expect(screen.getByText('Orders', { ignore: 'nav *' }));
   });
+
+  test('renders custom breadcrumbs', async () => {
+    render(
+      <PageContainer
+        breadcrumbs={[
+          { title: 'Hello', path: '/hello' },
+          { title: 'World', path: '/world' },
+        ]}
+      />,
+    );
+
+    const breadcrumbs = screen.getByRole('navigation', { name: 'breadcrumb' });
+
+    const helloLink = within(breadcrumbs).getByRole('link', { name: 'Hello' });
+    expect(helloLink.getAttribute('href')).toBe('/hello');
+    expect(within(breadcrumbs).getByText('World')).toBeTruthy();
+  });
+
+  // TODO: Remove in the next major version
```

Comment:
Should we remove it in the next minor itself and not wait for a major? Perhaps breaking changes are okay to be made in a minor while the project is in beta

Reply:
sure, but it's not even in the current minor yet